bibtex/zhang2021fast.bib

```diff
@@ -1,6 +1,8 @@
 @inproceedings{zhang2021fast,
 	title={Fast Multi-Resolution Transformer Fine-tuning for Extreme Multi-label Text Classification},
 	author={Zhang, Jiong and Chang, Wei-Cheng and Yu, Hsiang-Fu and Dhillon, Inderjit S},
-	journal={arXiv preprint arXiv:2110.00685},
+	journal={Advances in Neural Information Processing Systems},
```

Comment:
Should it be `booktitle` instead?  For example (https://openreview.net/forum?id=gjBz22V93a):
```
@inproceedings{
zhang2021fast,
title={Fast Multi-Resolution Transformer Fine-tuning for Extreme Multi-label Text Classification},
author={Jiong Zhang and Wei-Cheng Chang and Hsiang-Fu Yu and Inderjit S Dhillon},
booktitle={Advances in Neural Information Processing Systems},
editor={A. Beygelzimer and Y. Dauphin and P. Liang and J. Wortman Vaughan},
year={2021},
url={https://openreview.net/forum?id=gjBz22V93a}
}
```